src/muya/lib/parser/parse.js

```diff
@@ -16,21 +17,21 @@ const validWidthAndHeight = value => {
   return value >= 0 ? value : ''
 }
 
-const getSrcAlt = text => {
-  const SRC_REG = /src\s*=\s*("|')([^\1]+?)\1/
-  const ALT_REG = /alt\s*=\s*("|')([^\1]+?)\1/
-  const WIDTH_REG = /width\s*=\s*("|')([^\1]+?)\1/
-  const HEIGHT_REG = /height\s*=\s*("|')([^\1]+?)\1/
-  const srcMatch = SRC_REG.exec(text)
-  const src = srcMatch ? srcMatch[2] : ''
-  const altMatch = ALT_REG.exec(text)
-  const alt = altMatch ? altMatch[2] : ''
-  const widthMatch = WIDTH_REG.exec(text)
-  const width = widthMatch ? validWidthAndHeight(widthMatch[2]) : ''
-  const heightMatch = HEIGHT_REG.exec(text)
-  const height = heightMatch ? validWidthAndHeight(heightMatch[2]) : ''
+const getAttributes = html => {
+  const parser = new DOMParser()
+  const doc = parser.parseFromString(html, 'text/html')
+  const target = doc.querySelector('body').firstElementChild
+  if (!target) return null
+  const attrs = {}
+  for (const attr of target.getAttributeNames()) {
+    if (/width|height/.test(attr)) {
+      attrs[attr] = validWidthAndHeight(target.getAttribute(attr))
+    } else {
```

Comment:
`} else if (!global.MARKTEXT_SAFE_MODE) {`

If not `src`, `alt`, `width` or `height`.